src/serverlessFunctions/parseFromTemplate.js

```diff
@@ -23,8 +23,32 @@ const buildFnPathData = (path) => {
   };
 };
 
+const buildLayersConfig = (basePath, templatePath, template, prelimLayers) => {
+  const config = [];
+
+  if (prelimLayers && prelimLayers.length > 0) {
+    prelimLayers.forEach((layer) => {
+      if (!('Ref' in layer)) throw new Error('Only referenced layers are supported.');
```

Comment:
Won't this be a breaking change and a blocker for people using layers without `Ref`? 🤔 

I wonder if in these cases we should simply log a warning and skip the current layer being processed, what do you think?

```suggestion
      if (!('Ref' in layer)) {
        console.warn('Only referenced layers are supported, skipping layer', layer);
        return;
      }
```

Or, do you reckon this behavior could be misleading?

Reply:
That's a good catch. I agree.